weblate/static/editor/full.js

```diff
@@ -540,6 +562,17 @@
             "</td>"
         )
       );
+      if (el.delete_url) {
+        row.append(
+          $(
+            "<td>" +
+              '<a class="js-delete-machinery btn btn-danger">' +
+              gettext("Delete entry") +
+              "</a>" +
+              "</td>"
```

Comment:
You're adding another table column here, doesn't it break formatting?

Reply:
It does not apparently.
![Screenshot 2022-08-12 at 13 26 44](https://user-images.githubusercontent.com/24358501/184345021-1d7acdf2-6efe-4e77-87f6-cd94be348d0d.png)

Looks fine on my screen and I tried other big and a bit small screens. Please let me know if it looks good.